src/server/sockets/sockets.ts

```diff
@@ -127,6 +127,15 @@ export const configureIo = (server: HttpServer) => {
             ).emit('gameChatMessage', systemMessage);
         };
 
+    const displayLastPlayedCardForRoom = (socket: Socket) => (card: Card) => {
```

Comment:
This is a curried function that's basically a factory function for making other functions.

The reason we do it like this is so that for any given socket, we can get a function that takes in a card and does stuff with that card (in this case emitting it to all the players in the room)